.plzconfig

```diff
@@ -1,6 +1,3 @@
-[please]
-location = ./plz-out/please
```

Comment:
I remember this being added for ease of development, where you run `./bootstrap` and then `plz` would resolve correctly. It looks like that won't work anymore?

Reply:
Oh, does that actually work? That's neat. The reason we introduced this was so that `./plz-out/please/please_go` was on the path which was required for `//tools/please_go`. I can leave it in if it's helping with development.

Reply:
The built please will now be available in `plz-out/bin/src` locally which changes things a bit. It's not too bad, a `.plzconfig.local` file can be created for the same effect. It was just nice that you didn't need the extra step.